tests/shared/helpers.py

```diff
@@ -189,12 +189,16 @@ def dump_to_json(local_path: Path, data: Dict[str, np.ndarray]):
         json.dump(deepcopy(data), file, indent=4, cls=NumpyEncoder)
 
 
-def compare_stats(expected: Dict[str, np.ndarray], actual: Dict[str, np.ndarray], param_names: List[str]):
+def compare_stats(
+    expected: Dict[str, np.ndarray], actual: Dict[str, np.ndarray], param_names: List[str], strict: bool = True
```

Comment:
@kshpv, I'd like to use this function for int4-int8 quantization, but there's a limitation for getting u4 constant. Therefore, not all stats are available. With strict=False I was able to compare all stats for int8 layers, and part of stats for int4 ones.